store/tikv/2pc.go

```diff
@@ -615,8 +615,10 @@ func (c *twoPhaseCommitter) execute(ctx context.Context) error {
 		}
 	}()
 
+	commitTSFuture := c.store.getTimestampFuture(ctx)
 	binlogChan := c.prewriteBinlog()
```

Comment:
start fetch the commit ts before successfully write the Binlog will break the algorithm to sort the Binlog by commit ts which depend on you acquire the commit ts after successfully write the Binlog  @GregoryIan PTAL